client/Cargo.toml

```diff
@@ -72,6 +73,8 @@ tokio = { workspace = true, features = ["rt"] }
 tokio-tungstenite = { workspace = true }
 tungstenite = { workspace = true }
 futures-util = "0.3.28"
+merge = "0.1.0"
```

Comment:
this crate doesn't really do any work. You can implement this yourself

Reply:
This crate provides `Merge` procmacro to generate boilerplate. I don't want to bother with writing this manually at this stage of the work.

In the "Post-merge" section I've described that one of the next steps is to develop a generic configuration library. That library will contain an in-house macro for this purpose.

Reply:
> This crate provides Merge procmacro to generate boilerplate. I don't want to bother with writing this manually at this stage of the work.

on the other hand I don't want to see dependency hell. Boilerplate here is very miniscule.